dns/route53/client.go

```diff
@@ -1,14 +1,25 @@
 package route53
 
 import (
+	"fmt"
+	"time"
+
 	"github.com/aws/aws-sdk-go/aws"
 	awsSession "github.com/aws/aws-sdk-go/aws/session"
 	route53Client "github.com/aws/aws-sdk-go/service/route53"
 )
 
+// NOTE: the minimal interface to the route 53 client used for client mocking
```

Comment:
Please refer to https://blog.golang.org/godoc and start the comment with the name of the thing you're documenting.

Reply:
Adjusted.